Update GA4 plans (#2009)
segmentio/segment-docs: src/connections/destinations/catalog/google-analytics/ga4-plans.md

Google introduced the new version of Google Analytics, called Google Analytics 4 (GA4), in October 2020. GA4 has some distinct differences from Universal Analytics (UA), which are important to understand before considering migration and the data schema changes that might require.

> success ""

> > Segment expects to release a beta GA4 destination in Q4 2021. We’ll update this page to share the latest on our GA4 destination.

Reviewer: ```suggestion
> Segment expects to release a beta GA4 destination in Q4 2021. Segment will update this page to share the latest on the GA4 destination.
```
